bench/src/main/scala/Benchmarks.scala

```diff
@@ -30,8 +30,11 @@ object Bench {
     val iterations = if (intArgs.length > 1) intArgs(1).toInt else 20
     val forks = if (intArgs.length > 2) intArgs(2).toInt else 1
 
+
+    import File.{ separator => sep }
```

Comment:
Import once at top level?

Reply:
The two places are using different: one is using `File. separator`, while the other is using `File.pathSeparator`.